### ISSUE TITLE
Uncouple blob limits per block across CL and EL

### ISSUE THREAD

**Issue by zilm13** [Nashatyrev/artemis#84]
PR: https://github.com/ethereum/consensus-specs/pull/3800
EIP: https://eips.ethereum.org/EIPS/eip-7742
Wait for merge and implement 
Plus we don't need this until we have at least 1 EL which supports this

**Comment by lucassaldanha** [comment]
Could probably close this one in favour of https://github.com/Consensys/teku/issues/8745

**Comment by zilm13** [comment]
yeah I see it's delayed, closing this one, it's easier to open new one when EIP-7742 is closer to the production
